executor/simple.go

```diff
@@ -1031,6 +1382,25 @@ func (e *SimpleExec) executeAlterUser(ctx context.Context, s *ast.AlterUserStmt)
 		}
 	}
 
+	sysSession, err := e.getSysSession()
+	defer clearSysSession(ctx, sysSession)
+	if err != nil {
+		return err
+	}
+	sqlExecutor := sysSession.(sqlexec.SQLExecutor)
+	// session isolation level changed to  READ-COMMITTED
```

Comment:
```suggestion
	// session isolation level changed to READ-COMMITTED
```
Is there any specific reason to do this?